eclair-core/src/test/scala/fr/acinq/eclair/channel/states/StateTestsHelperMethods.scala

```diff
@@ -29,16 +29,22 @@ trait StateTestsHelperMethods extends TestKitBase {
                    router: TestProbe,
                    relayer: TestProbe)
 
-  def init(): Setup = {
+  def init(chainHash_opt: Option[BinaryData] = None): Setup = {
```

Comment:
`init` should take `nodeParamsA` a,d `nodeParamsB` as parameters, with default to `TestConstants.Alice.nodeParams` and `TestConstants.Bob.nodeParams`